### ISSUE TITLE
VF installer on Windows 

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#2639]
### Observed behavior

1. No issues during installation itself nor the Setup Wizard
2. Channels import OK
3. `500 Server error` when I try to see the user account, so I stop the server and run `C:\Python27\Scripts>kolibri plugin kolibri.plugins.user disable`
4. Now I get the 404 on user account page
![win7 start running - oracle vm virtualbox_047](https://user-images.githubusercontent.com/1457929/33032599-fe4efad4-ce21-11e7-9ab0-2a817dfe9a63.png)
5. I try running 
```setx DJANGO_SETTINGS_MODULE "kolibri_instant_schools_plugin.instant_schools_settings"```, and now I can finally see the the admin account page correctly
6. Now I logout and try to login with the admin creds created with the setup wizard and it fails. 
7. Then I run `C:\Python27\Scripts>kolibri manage migrate --database=instant_schools` and restart the server.
```
C:\Python27\Scripts>kolibri manage migrate --database=instant_schools
Operations to perform:
  Apply all migrations: sessions, admin, kolibriauth, morango, auth, content, ko
libri_instant_schools_plugin, contenttypes, exams, device, logger
Running migrations:
  Rendering model states... DONE
  Applying kolibriauth.0001_initial... OK
  Applying contenttypes.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0001_initial... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying content.0001_initial... OK
  Applying content.0002_channelmetadatacache_last_updated... OK
  Applying content.0003_auto_20170607_1212... OK
  Applying content.0004_auto_20170825_1038... OK
  Applying content.0005_auto_20171009_0903... OK
  Applying kolibriauth.0002_auto_20170608_2125... OK
  Applying kolibriauth.0003_auto_20170621_0958... OK
  Applying device.0001_initial... OK
  Applying exams.0001_initial... OK
  Applying kolibri_instant_schools_plugin.0001_initial... OK
  Applying kolibri_instant_schools_plugin.0002_import_old_hashed_mappings... OK
  Applying kolibri_instant_schools_plugin.0003_passwordresettoken... OK
  Applying kolibriauth.0004_auto_20170816_1607... OK
  Applying kolibriauth.0005_auto_20170818_1203... OK
  Applying logger.0001_initial... OK
  Applying logger.0002_auto_20170518_1031... OK
  Applying logger.0003_auto_20170531_1140... OK
  Applying morango.0001_initial... OK
  Applying morango.0002_auto_20170511_0400... OK
  Applying morango.0003_auto_20170519_0543... OK
  Applying morango.0004_auto_20170520_2112... OK
  Applying morango.0005_auto_20170629_2139... OK
  Applying morango.0006_instanceidmodel_system_id... OK
  Applying sessions.0001_initial... OK
```

8. Since I still cannot login with the admin account created previously, I try to create another:

```
C:\Python27\Scripts>kolibri manage createsuperuser
Username: 12345678901
Password:
Password (again):
Superuser created successfully.

C:\Python27\Scripts>kolibri start --debug --foreground
?[01;30mDEBUG    Debug mode is on!?[0m
?[37mINFO     Running update routines for new version...?[0m

0 static files copied to 'C:\Users\IEUser\.kolibri\static', 406 unmodified.
Operations to perform:
  Apply all migrations: sessions, admin, kolibriauth, morango, auth, content, ko
libri_instant_schools_plugin, contenttypes, exams, device, logger
Running migrations:
  No migrations to apply.
  Your models have changes that are not yet reflected in a migration, and so won
't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage
.py migrate' to apply them.
?[37mINFO     Running 'kolibri start' in foreground...?[0m
?[37mINFO     Kolibri running on:
?[0m
        http://10.0.2.15:8080/
```

But I still cannot login with the superuser created from the command prompt:

![win7 start running - oracle vm virtualbox_048](https://user-images.githubusercontent.com/1457929/33033432-a1ad1e7a-ce24-11e7-8d1c-facdae773643.png)

9. However I can create a new user from the `Create Account` button, login without problems and browse content

### Context

* Kolibri version: `KolibriSetup-0.6.1.dev020171118184434_git.exe`
* Operating system: Windows 7
* Browser: Chrome

cc @rtibbles

**Comment by indirectlylit** [comment]
as part of this task, let's make a new windows 1.1.0 installer and pin to that

**Comment by radinamatic** [comment]
We already mentioned that with @rtibbles .

And the installer is looking better, only the `manage migrate --database=instant_schools` is still pending. 

![win7 start running - oracle vm virtualbox_052](https://user-images.githubusercontent.com/1457929/33082023-0d0e28dc-cedc-11e7-892a-36aaddc356c9.png)

**Comment by rtibbles** [comment]
Fixed in #2633 and #2650.
